sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -181,6 +190,30 @@ def plot_partial_dependence(estimator, X, features, *, feature_names=None,
 
         .. versionadded:: 0.22
 
+    kind : {'average', 'individual', 'both'}, default='average'
+        Whether to plot the partial dependence averaged across all the samples
+        in the dataset or one line per sample or both.
+
+        - kind='average' results in the traditional PD plot;
+        - kind='individual' results in the ICE plot.
+
+       Note that the fast method='recursion' option is only available for
+       kind='average'. Plotting individual dependencies requires using the
+       slower method='brute' option.
+
+        .. versionadded:: 0.23
+
+    subsample : float, int or None, default=1000
+        Sampling for ICE curves when `kind` is 'individual' or 'both'.
+        If `float`, should be between 0.0 and 1.0 and represent the proportion
+        of the dataset to be used to plot ICE curves. If `int`, represents the
+        absolute number samples to use.
+
+        Note that the full dataset is still used to calculate partial
```

Comment:
to calculate the "averaged" partial

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/ec18fd72861f90fa6c87151de9c885ea6ee964a1